lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -451,11 +452,26 @@ private boolean achievedWithGpg45Profile(
                         ? gpg45ProfileEvaluator.getFirstMatchingProfile(
                                 gpg45Scores, requestedVot.getSupportedGpg45Profiles())
                         : Optional.empty();
+        List<SignedJWT> gpg45Credentials = new ArrayList<>();
+        for (SignedJWT credential : credentials) {
+            if (!VcHelper.isOperationalProfileVc(credential)) {
+                gpg45Credentials.add(credential);
+            }
+        }
```

Comment:
We've got a helper for this already - `VcHelper.filterVCBasedOnProfileType`

Reply:
True, but this takes `VcStoreItem` instead of the `SignedJWT` we are trying to filter at this stage. 

This would mean creating another instance of `VcHelper.filterVCBasedOnProfileType` to handle `SignedJWT`.

Reply:
Ugh, yeah the difference between `SignedJWT` and `VCStoreItem` is awkward. I'm hoping we can fix this as part of PYIC-3908, but we might have to do some groundwork first.

If you can't see a neat fix then perhaps this is ok for now.